src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenCheckoutApiClient.java

```diff
@@ -0,0 +1,39 @@
+package org.killbill.billing.plugin.adyen.client.payment.service;
+
+import com.adyen.Client;
+import com.adyen.enums.Environment;
+import com.adyen.model.checkout.PaymentsDetailsRequest;
+import com.adyen.model.checkout.PaymentsRequest;
+import com.adyen.model.checkout.PaymentsResponse;
+import com.adyen.service.Checkout;
+import com.adyen.service.exception.ApiException;
+import org.jooq.tools.StringUtils;
+import org.killbill.billing.plugin.adyen.client.AdyenConfigProperties;
+
+import java.io.IOException;
+
+public class AdyenCheckoutApiClient {
+    final Checkout checkoutApi;
+
+    public AdyenCheckoutApiClient(final AdyenConfigProperties adyenConfigProperties) {
+        // initialize the REST client here
+        Environment environment = Environment.TEST; //default environment
```

Comment:
Can you please add more detail. Looks like this TEST points to Adyen Test API endpoint. 
Can you please make this clear in the comment. As we have Staging and Dev environments for testing, and it will confuse developer.